packages/apps/reputation-oracle/server/src/modules/credentials/credential.entity.ts

```diff
@@ -0,0 +1,44 @@
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { NS } from '../../common/constants';
+import { BaseEntity } from '../../database/base.entity';
+import { CredentialStatus } from '../../common/enums/credential';
+import { UserEntity } from '../user/user.entity';
+
+@Entity({ schema: NS, name: 'credential' })
```

Comment:
Can you set the name of the table in plural? To match the style of the other tables.

Reply:
done